crates/core/tedge/src/cli/connect/bridge_config_aws.rs

```diff
@@ -2,6 +2,8 @@ use crate::cli::connect::BridgeConfig;
 use camino::Utf8PathBuf;
 use tedge_config::ConnectUrl;
 
+const MOSQUITTO_BRIDGE_TOPIC: &str = "te/device/main/service/mosquitto-aws-bridge/status/health";
```

Comment:
What will happen if the `mosquitto` runs on a separate device? is it still considered a service of the main device?

Reply:
Obviously this `tedge connect` logic won't work if mosquitto is running on another device as we don't support updating the bridge config of a remote broker.

But even ignoring that fact, this cloud bridge is something that is associated with the main device only. Even if the main device uses mosquitto running on another device, from the child's perspective, it's the main device's broker.

Now, the child devices may have their own brokers connected/bridged to the main device broker. But, even in that case, this `tedge connect` command is only interested in the main device's broker.